LongWarOfTheChosen/Src/LW_Overhaul/Classes/X2Ability_LW_AssaultAbilitySet.uc

```diff
@@ -635,3 +639,154 @@ static function X2AbilityTemplate CreateChainLightningAbility()
+	Template.AbilityMultiTargetStyle = new class'X2AbilityMultiTarget_Volt';
+	Template.AbilityTriggers.AddItem(default.PlayerInputTrigger);
+	Template.TargetingMethod = class'X2TargetingMethod_Volt';
+
+	Template.AbilityShooterConditions.AddItem(default.LivingShooterProperty);
+	Template.AddShooterEffectExclusions();
+
+	Template.AbilityTargetConditions.AddItem(default.GameplayVisibilityCondition);
+	Template.AbilityTargetConditions.AddItem(default.LivingHostileUnitDisallowMindControlProperty);
+
+
+	//Template.bAllowAmmoEffects = true;
+	Template.bAllowBonusWeaponEffects = true;
+
+	// Can't target dead; Can't target friendlies -- must be enemy organic
+	UnitPropertyCondition = new class'X2Condition_UnitProperty';
+	UnitPropertyCondition.ExcludeRobotic = true;
+	UnitPropertyCondition.ExcludeOrganic = false;
+	UnitPropertyCondition.ExcludeDead = true;
+	UnitPropertyCondition.ExcludeFriendlyToSource = true;
+	UnitPropertyCondition.RequireWithinRange = true;
+	Template.AbilityTargetConditions.AddItem(UnitPropertyCondition);
+	
+	UnitPropertyCondition2=new class'X2Condition_UnitProperty';
+	UnitPropertyCondition2.ExcludeConcealed = true;
+	Template.AbilityShooterConditions.AddItem(UnitPropertyCondition2);
+
+	SuppressedCondition = new class'X2Condition_UnitEffects';
+	SuppressedCondition.AddExcludeEffect(class'X2Effect_Suppression'.default.EffectName, 'AA_UnitIsSuppressed');
+	SuppressedCondition.AddExcludeEffect(class'X2Effect_AreaSuppression'.default.EffectName, 'AA_UnitIsSuppressed');
+	Template.AbilityShooterConditions.AddItem(SuppressedCondition);
+
+	ImmuneUnitCondition = new class'X2Condition_UnitType';
+	ImmuneUnitCondition.ExcludeTypes.AddItem('PsiZombie');
+	ImmuneUnitCondition.ExcludeTypes.AddItem('AdvPsiWitchM2');
+	ImmuneUnitCondition.ExcludeTypes.AddItem('AdvPsiWitchM3');
+	Template.AbilityTargetConditions.AddItem(ImmuneUnitCondition);
+
+
+
+	Cooldown = new class'X2AbilityCooldown';
+	Cooldown.iNumTurns = default.CHAIN_LIGHTNING_COOLDOWN;
+	Template.AbilityCooldown = Cooldown;
+
+	// Action Point
+	ActionPointCost = new class'X2AbilityCost_ActionPoints';
+	ActionPointCost.iNumPoints = default.CHAIN_LIGHTNING_MIN_ACTION_REQ;
+	ActionPointCost.DoNotConsumeAllSoldierAbilities.AddItem('Unlimitedpower_LW');
+	ActionPointCost.bConsumeAllPoints = true;
+	Template.AbilityCosts.AddItem(ActionPointCost);	
+
+	//Stun Effect
+	StunnedEffect = CreateArcthrowerStunnedStatusEffect(100);
+	Template.AddTargetEffect(StunnedEffect);
+	Template.AddMultiTargetEffect(StunnedEffect);
+
+	//CleanUpEffect = CreateStunnedEffectsCleanUpEffect();
+	//Template.AddTargetEffect(CleanUpEffect);
+	DamageEffect = new class'X2Effect_ApplyWeaponDamage';
+	DamageEffect.bIgnoreBaseDamage = true;
+	DamageEffect.DamageTag = 'Volt';
+	DamageEffect.bIgnoreArmor = true;
+	DamageEffect.TargetConditions.AddItem(UnitPropertyCondition);
+	Template.AddTargetEffect(DamageEffect);
+	Template.AddMultiTargetEffect(DamageEffect);
```

Comment:
Why is this doing Volt damage?

Reply:
Trial and error for targeting, probably unneeded.